x-pack/solutions/observability/plugins/infra/public/pages/metrics/index.tsx

```diff
@@ -93,15 +93,15 @@ export const InfrastructurePage = () => {
                         <EuiHeaderLink color={'text'} {...settingsLinkProps}>
                           {settingsTabTitle}
                         </EuiHeaderLink>
-                        <Routes>
+                        <Routes enableExecutionContextTracking={true}>
```

Comment:
I think we're not interested in enabling it here. This is just to show/hide the header action menu, based on the current route

Reply:
Removed in [abcd707](https://github.com/elastic/kibana/pull/204939/commits/abcd7070c0044fefef37e7c863183a7f8f7019fc)